aspnetcore/web-api/index.md

```diff
@@ -867,238 +881,3 @@ The `[Consumes]` attribute is applied to both actions. The `PostJson` action han
 * [Microsoft Learn: Create a web API with ASP.NET Core](/learn/modules/build-web-api-aspnet-core/)
 
 :::moniker-end
-
-:::moniker range="< aspnetcore-3.0"
```

Comment:
Can you add `:::moniker range="< aspnetcore-3.0` back in? We've been told not to remove old versions.